distributed/wiki/local/index.yml

```diff
@@ -1,39 +1,22 @@
 jtype: Flow
 version: '1'
 with:
-  rest_api: {{ JINA_GATEWAY_REST }}
-  port_expose: {{ JINA_GATEWAY_PORT_EXPOSE }}
+  workspace: $JINA_WORKDIR
+  py_modules:
+    - wiki_executors.py
 pods:
   - name: segmenter
-    polling: any
-    shards: {{ JINA_SEGMENTER_SHARDS }}
```

Comment:
why are we removing the sharding? Testing this as part of the stress test is one of the added values here I think?

Reply:
same as above

Reply:
Maybe I should convert this PR to a draft one.
Let me fix the status of this PR.